aerospike-storage-backend/src/main/java/com/playtika/janusgraph/aerospike/wal/WriteAheadLogCompleter.java

```diff
@@ -83,13 +93,13 @@ private void completeHangedTransactions(){
                                 transaction.transactionId, transaction.locks, transaction.mutations, true);
                         logger.info("Successfully complete transaction id={}", transaction.transactionId);
                     }
-
                     //this is expected behaviour that may have place in case of transaction was interrupted:
                     // - on 'release locks' stage then transaction will fail and just need to release hanged locks
                     // - on 'delete wal transaction' stage and just need to remove transaction
-                    catch (BackendException be) {
-                        logger.info("Failed to complete transaction id={}",
-                                transaction.transactionId, be);
+                    catch (PermanentLockingException be) {
+                        logger.info("Failed to complete transaction id={} as it's already completed", transaction.transactionId, be);
```

Comment:
if this is ok case why do we need to log that on info level? debug should be enough then

Reply:
yes. It's expected but quite rare corner case. So want to log it with info level to know the frequency of this cases.